shared/src/search/suggestions/util.ts

```diff
@@ -1,24 +1,14 @@
-export enum SuggestionTypes {
+import { FilterTypes } from '../interactive/util'
+
+export type SuggestionTypes = FilterTypes | NonFilterSuggestionTypes
+
+/**
+ * NonFilterSuggestionTypes represents the types of suggestion results that do not match a filter.
+ *
+ * For example, there is no `symbol:` filter, but there are symbol suggestion results.
+ */
+export enum NonFilterSuggestionTypes {
```

Comment:
Could we take this opportunity to make this name (as well as `SuggestionTypes`, `FilterTypes`, `FiltersSuggestionTypes`) singular? They are the only pluralized enum / union type names in our codebase and throw me off every time 😅 

```suggestion
export enum NonFilterSuggestionType {
```